### ISSUE TITLE
Orderer check via logs is not robust

### ISSUE THREAD

**Issue by nicolapaoli** [hyperledger-labs/nephos#29]
After deploying a network and leaving it for some time without activity, the orderer log that check it is functioning stalls.

**Comment by inzamam-iqbal** [comment]
Hyperledger 1.4 peer and orderer supports [helath checks ](https://hyperledger-fabric.readthedocs.io/en/release-1.4/operations_service.html#health-checks) where we can just get the health status by sending a GET request to `/healthz`. So utilizing that feature after updating to hyperladger 1.4 will be the best solution for this.

**Comment by alexvicegrab** [comment]
Agreed, I would suggest to add these health checks, and if we don't find the `/healthz`, to default to the old code, but add a deprecation notice. This would mention that in 3/6/12 months (I favour 6, but we can perhaps vote on this?), we will stop supporting Fabric >1.4, so that this change does not come as a surprise.
